internal/scorecard/config.go

```diff
@@ -49,8 +49,8 @@ func (e externalPluginEnv) String() string {
 	return fmt.Sprintf("%s=%s", e.Name, e.Value)
 }
 
-// validateConfig takes a viper config for a plugin and returns a nil error if valid or an error explaining why the config is invalid
-func validateConfig(config pluginConfig, idx int, version string) error {
+// validateConfig takes a config for a plugin and returns a nil error if valid or an error explaining why the config is invalid
+func validateConfig(config PluginConfig, idx int, version string) error {
```

Comment:
it now is with uppercase. 
Is it been used in another place? If not, I think that is better keep pluginConfig.

Reply:
its exposed now so that the cmd package can access it.